codespell_lib/data/dictionary.txt

```diff
@@ -14250,11 +14255,13 @@ massmedia->mass media
 masster->master
 masteer->master
 masterbation->masturbation
+mata->meta
```

Comment:
Probably also suggest mater, in the alma mater sense: https://en.wikipedia.org/wiki/Alma_mater

Reply:
b770734